Tangem/App/Services/Analytics/Analytics+ParameterKey.swift

```diff
@@ -74,5 +74,8 @@ extension Analytics {
         case result = "Result"
         case input = "Input"
         case passphrase = "Passphrase"
+
+        case fromSummary = "FromSummary"
```

Comment:
кажется что так должно быть:
```suggestion
        case fromSummary = "From Summary"
```

Reply:
Точно, спасибо